autospec/build.py

```diff
@@ -331,29 +331,49 @@ def package(filemanager, mockconfig, mockopts, cleanup=False):
         shutil.rmtree('{}/results'.format(download_path), ignore_errors=True)
         os.makedirs('{}/results'.format(download_path))
 
-    util.call("{} -r {} --buildsrpm --sources=./ --spec={}.spec "
-              "--uniqueext={} --result=results/ {} {}"
-              .format(mock_cmd, mockconfig, tarball.name, uniqueext, cleanup_flag,
-                      mockopts),
-              logfile="%s/results/mock_srpm.log" % download_path, cwd=download_path)
+    cmd_args = [
+        mock_cmd,
+        f"--root={mockconfig}",
+        "--buildsrpm",
+        "--sources=./",
+        f"--spec={tarball.name}.spec",
+        f"--uniqueext={uniqueext}",
+        "--result=results/",
+        cleanup_flag,
+        mockopts,
+    ]
+    util.call(" ".join(cmd_args),
```

Comment:
I'm a bit sad it has to be this way since call is just undoing the join but oh well. Maybe we could check the type arg for the command but meh.

Reply:
We could require `util.call()` to have a list for the first argument. That would be my preference.